lib/vagrant-parallels/action/sane_defaults.rb

```diff
@@ -28,21 +28,23 @@ def call(env)
 
         def default_settings
           {
-            tools_autoupdate: 'no',
-            on_shutdown: 'close',
-            on_window_close: 'keep-running',
-            auto_share_camera: 'off',
-            smart_guard: 'off',
-            longer_battery_life: 'on',
-            shared_cloud: 'off',
-            shared_profile: 'off',
-            smart_mount: 'off',
-            sh_app_guest_to_host: 'off',
-            sh_app_host_to_guest: 'off',
-            startup_view: 'headless',
-            time_sync: 'on',
-            disable_timezone_sync: 'on',
-            shf_host_defined: 'off'
+            # all commented for macOS VM, need a better solution here
+            
+            # tools_autoupdate: 'no',
+            # on_shutdown: 'close',
+            # on_window_close: 'keep-running',
+            # auto_share_camera: 'off',
+            # smart_guard: 'off',
+            # longer_battery_life: 'on',
+            # shared_cloud: 'off',
+            # shared_profile: 'off',
+            # smart_mount: 'off',
+            # sh_app_guest_to_host: 'off',
+            # sh_app_host_to_guest: 'off',
+            # startup_view: 'headless',
+            # time_sync: 'on',
+            # disable_timezone_sync: 'on',
+            # shf_host_defined: 'off'
```

Comment:
**question:** Are all of these settings not supported for "macvm" machines? 

In this case the better solution would be:
```ruby
def default_settings
  # Options defined below are not supported for `*.macvm` VMs
  return {} if is_macvm(@env)

  {
    tools_autoupdate: 'no',
    on_shutdown: 'close',
    on_window_close: 'keep-running',
    auto_share_camera: 'off',
    smart_guard: 'off',
    longer_battery_life: 'on',
    shared_cloud: 'off',
    shared_profile: 'off',
    smart_mount: 'off',
    sh_app_guest_to_host: 'off',
    sh_app_host_to_guest: 'off',
    startup_view: 'headless',
    time_sync: 'on',
    disable_timezone_sync: 'on',
    shf_host_defined: 'off'
  }
end
```